specification/synapse/data-plane/Microsoft.Synapse/preview/2020-10-01-preview/monitoring.json

```diff
@@ -0,0 +1,651 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SQLPoolName"
+          },
+          {
+            "$ref": "#/parameters/SQLDatabaseName"
+          },
+          {
+            "$ref": "#/parameters/SQLStreamingJobName"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "type": "string",
+            "description": "The **$filter** is used to reduce the set of metric data returned. Example: Metric contains metadata A, B and C. - Return all time series of C where A = a1 and B = b1 or b2 **$filter=A eq 'a1' and B eq 'b1' or B eq 'b2' and C eq '*'** - Invalid variant: **$filter=A eq 'a1' and B eq 'b1' and C eq '*' or B = 'b2'** This is invalid because the logical or operator cannot separate two different metadata names. - Return all time series where A = a1, B = b1 and C = c1: **$filter=A eq 'a1' and B eq 'b1' and C eq 'c1'** - Return all time series where A = a1 **$filter=A eq 'a1' and B eq '*' and C eq '*'**. Special case: When dimension name or dimension value uses round brackets. Eg: When dimension name is **dim (test) 1** Instead of using $filter= \"dim (test) 1 eq '*' \" use **$filter= \"dim %2528test%2529 1 eq '*' \"** When dimension name is **dim (test) 3** and dimension value is **dim3 (test) val** Instead of using $filter= \"dim (test) 3 eq 'dim3 (test) val' \" use **$filter= \"dim %2528test%2529 3 eq 'dim3 %2528test%2529 val' \"**",
+            "required": false
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/TimespanParameter"
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/IntervalParameter"
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/MetricNamesParameter"
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/AggregationsParameter"
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/OrderByParameter"
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/ResultTypeParameter"
+          },
+          {
+            "$ref": "../../../../../monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json#/parameters/MetricNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
```

Comment:
How about defining them in this Swagger spec rather than taking dependencies on Swagger of other serice?